mock/mock.go

```diff
@@ -171,6 +187,7 @@ type Service struct {
 	RawStatePayload             *light.RawStatePayload
 	RawStateDeviceChainPayload  *tile.RawStateDeviceChainPayload
 	RawStateTileState64Payloads []*tile.RawStateTileState64Payload
+	RawEchoResponsePayload      *lifxlan.RawEchoResponsePayload
```

Comment:
This field is unused.

Other `Raw*Payload` fields defined here are the *fixed* payload to be responded when the request comes in. For echo the response is never fixed, we just return whatever comes in back, so there's no need to define this field here.

Reply:
Whoops, forgot to delete that.